.github/workflows/pkgci_test_amd_mi300.yml

```diff
@@ -19,7 +19,10 @@ on:
 
 jobs:
   test_mi300:
-    runs-on: nodai-amdgpu-mi300-x86-64
+    runs-on: linux-mi300-gpu-1
+    container:
+      image: rocm/dev-ubuntu-22.04:6.3
+      options: --user root --device=/dev/kfd --device=/dev/dri --ipc=host --group-add video --cap-add=SYS_PTRACE --security-opt seccomp=unconfined
```

Comment:
This had a failure after merge:
```
Error response from daemon: error gathering device information while adding custom device "/dev/kfd": no such file or directory
  Error: failed to start containers: 4bec79a3d99b9a5f5195ceb3ae3bf4ebb508b078b46d1de1d9b0366f0be182[41](https://github.com/iree-org/iree/actions/runs/12751728442/job/35540191551#step:2:44)
```

https://github.com/iree-org/iree/actions/runs/12751728442/job/35540191551

We can retry the job if you think that's a flake. Otherwise, want to revert for now?

Reply:
A retry succeeded 🤔